manifests/config.pp

```diff
@@ -45,46 +43,37 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0644',
-    content => template('icinga/defaults.erb'),
+    content => template('icinga/etc/default/icinga.erb'),
     notify  => [Class[icinga::service],Class[icinga::idoservice]],
     require => Class[icinga::install],
   }
 
-  file { '/etc/init.d/icinga':
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0755',
-    content => template('icinga/init.erb'),
-    notify  => Class[icinga::service],
-    require => Class[icinga::install],
-  }
-
   file { '/etc/icinga/icinga.cfg':
```

Comment:
There may have been a reason I was managing this but I can't think of it now.
It's best we don't if we are supporting $::osfamily == RedHat as well. :)